src/components/Auth/TodosItem.tsx

```diff
@@ -0,0 +1,110 @@
+import React, { memo, useState, useRef } from 'react';
+import cn from 'classnames';
+import { Todo } from '../../types/Todo';
+
+type Props = {
+  todo: Todo;
+  onDeleteTodo?: (todoId: number) => Promise<void>;
+  isDeleted?: boolean;
+  onChangeTodo?: (todoId: number, dataToUpdate: Partial<Todo>) => Promise<void>;
+};
+
+export const TodosItem: React.FC<Props> = memo(({
+  todo,
+  onDeleteTodo = () => {},
+  isDeleted,
+  onChangeTodo = () => {},
+}) => {
+  const todoTitleField = useRef<HTMLInputElement>(null);
```

Comment:
This field should be on focus when it appears in the page

Reply:
done